Openshift-EE/pipelines/OpenEBS-base/stages/2-setup/BDD/cstor/snapshot/cstor-snapshot-provisioning

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+set -x
+pod() {
+sshpass -p $pass ssh -o StrictHostKeyChecking=no $user@$ip -p $port 'cd e2e-openshift && GOPATH="/home/devuser/go"; PATH=$PATH:"/usr/local/go/bin:$GOPATH/bin"; /bin/bash /home/devuser/e2e-openshift/Openshift-EE/pipelines/OpenEBS-base/stages/2-setup/BDD/cstor/snapshot/cstor-snapshot-provisioning '"'$CI_JOB_ID'"'' '"'$CI_PIPELINE_ID'"' '"'$CI_COMMIT_SHA'"'
+}
+
+node() {
+
+job_id=$(echo $1) #Gitlab job id obtain from the gitlab env ($CI_JOB_ID).
+pipeline_id=$(echo $2)  # Gitlab pipeline id obtained from gitlab env ($CI_PIPELINE_ID).
+case_id=HC5P          # This need to be changed
+commit_id=$(echo $3) #Gitlab commit id Obtained fron gilab env ($CI_COMMIT_SHA).
+
+source ~/.profile
+
+#github token to push the test result into github mayadata-io/e2e-openshift repository.
+#This token is set as an env in ~/.profile in the test cluster.
+
+gittoken=$(echo "$github_token")
+
+time="date"
+current_time=$(eval $time)
+
+#pooling over previous job to complete
+bash Openshift-EE/utils/pooling jobname:bdd-cstor-volume-clone-provisioning
+bash Openshift-EE/utils/e2e-cr jobname:bdd-cstor-snapshot-provisioning jobphase:Running init_time:"$current_time" init_time:"$current_time" jobid:"$job_id" pipelineid:"$pipeline_id" testcaseid:"$case_id"
+
+cd /home/devuser/go/src/github.com/openebs
+
+# Running BDD test for cstor-snapshot-provisioning
+cd maya/tests/cstor/snapshot && ginkgo -v -- -kubeconfig=$HOME/.kube/config --cstor-replicas=3 --cstor-maxpools=3
+
+#Evaluating result based on the return count
+
+if [ "$?" != "0" ];then
+  echo "cstor-snapshot-provisioning-test-failed"
+  current_time=$(eval $time)
+  cd /home/devuser/e2e-openshift
+  bash Openshift-EE/utils/e2e-cr jobname:bdd-cstor-snapshot-provisioning jobphase:Completed end_time:"$current_time" jobid:"$job_id" pipelineid:"$pipeline_id" testcaseid:"$case_id" test_result:Fail
+  exit 1;
+else
+  echo "cstor-snapshot-provisioning-test-failed"
```

Comment:
@shashank855 test status failed or pass?

Reply:
@nsathyaseelan I have made the change.